src/canvas_backend/src/api.rs

```diff
@@ -31,6 +36,12 @@ pub struct Color {
     pub a: u8,
 }
 
+#[candid_method(query)]
+#[query]
+pub fn time_left() -> u64 {
+    storage::get::<EditsState>().time_left()
+}
+
```

Comment:
this is not generated in candid for some reason...

Reply:
I've been manually updating the candid by calling the tmp hack from dfx